typings/react-datetime-tests.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from 'react';
+import { Moment } from 'moment';
+import ReactDatetime = require('react-datetime');
```

Comment:
Sorry it took so long to reply, but I wanted to make sure I had it right. They are not equivalent.

The following is for CommonJS modules that use `module.exports = Thing`. This may be an ugly way of importing, but it is not within the scope of this PR to change it. It would also massively break the way non-typescript users require the module.

```ts
import ReactDatetime = require('react-datetime');
```

When you have multiple named exports `module.exports = { Thing: Thing }` or `export  class Thing { ... }` it is imported like so.

```ts
import * as ReactDatetime from 'react-datetime';
```

Finally, when you have a default export: `export default class Thing {...}`

```ts
import ReactDatetime from 'react-datetime';
```

You can read more here
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require